### PR TITLE
fixed the skeleton for staking contract, pushing for review

### PR DESCRIPTION
**Pull request by OkoliEvans**
@tensojka Please take a look

### CODE REVIEW

File: src/staking.cairo [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    struct CARM_Address_Set {
+        owner: ContractAddress,
+        CARM_token_address: ContractAddress
+    }
+
+    #[derive(starknet::Event, Drop)]
+    struct veCARM_Address_Set {
+        owner: ContractAddress,
+        veCARM_token_address: ContractAddress
+    }
```
Comment (tensojka) [inline src/staking.cairo:50]: I don't think these are necessary. Also, no need to emit the owner in this case.

Comment (OkoliEvans) [reply src/staking.cairo:50]: Cool, okay. I will make necessary adjustments and start off

**Comment by OkoliEvans** [comment]
Not tested yet, and hoping for some corrections after review. Will start with testing tonight @tensojka

**Comment by OkoliEvans** [comment]
made some updates to the contract, still testing though

**Pull request closed without merging**